ui/src/admin/components/chronograf/AllUsersTableRowNew.js

```diff
@@ -55,12 +54,12 @@ class AllUsersTableRowNew extends Component {
 
   handleSelectOrganization = newOrganization => {
     const newRoles = [
-      newOrganization.id === null
+      newOrganization.id === undefined
```

Comment:
consider comparing newOrganization to null organization?

Reply:
See above